signature/signer.go

```diff
@@ -0,0 +1,134 @@
+package signature
```

Comment:
I posted a comment on https://github.com/notaryproject/notation/discussions/278. Can you respond to that before I can review this code?

Reply:
Thanks for your comment, I have replied it.